zkevm-circuits/src/pi_circuit.rs

```diff
@@ -311,6 +342,11 @@ pub struct PiCircuitConfig<F: Field> {
     tx_table: TxTable,
     keccak_table: KeccakTable,
 
+    l1_block_hashes_count: Column<Advice>, // RLC(rpi) as the input to Keccak table
+    num_all_l1_block_hashes: Column<Advice>,
+    last_applied_l1_block: Column<Advice>,
+    last_applied_l1_block_from_last_block: Column<Advice>,
```

Comment:
by `*_from_last_block` do you mean from the last chunk? Comments would be good to have for each of these advice columns. Thanks :)